docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: "3.7"
```

Comment:
Not a meaningful change; this is deprecated anyway, so I'd just either leave the file untouched or fix it up properly

Reply:
was a change done by my auto formatter, will revert it.